src/core/Akka.Cluster.Tests/Akka.Cluster.Tests.csproj

```diff
@@ -3,7 +3,7 @@
 
   <PropertyGroup>
     <AssemblyTitle>Akka.Cluster.Tests</AssemblyTitle>
-    <TargetFrameworks>$(NetFrameworkTestVersion);$(NetCoreTestVersion)</TargetFrameworks>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
Same feedback as above - needs to be parameterized from `common.props` and kept with `<TargetFrameworks>` for all tests.